experimenter/tests/integration/nimbus/conftest.py

```diff
@@ -274,7 +274,7 @@ def _create_experiment(selenium, is_rollout=False):
         metrics = branches.save_and_continue()
         if default_data.metrics.primary_outcomes:
             metrics.set_primary_outcomes(values=default_data.metrics.primary_outcomes[0])
-            assert metrics.primary_outcomes.text != "", "The primary outcome was not set"
+            assert metrics.primary_outcomes[0] != "", "The primary outcome was not set"
```

Comment:
see here removed `.text`, if you will see circle ci- https://app.circleci.com/pipelines/github/mozilla/experimenter/21375/workflows/45ac91c3-9184-4c31-9a9a-4b0bd9383c60/jobs/67491 it is telling you exactly that use are using list and then you are still calling.text, how will that work,

Reply:
@yashikakhurana Thank you for the heads up. I will work on it asap